modules/andes-core/client/src/main/java/org/wso2/andes/jndi/PropertiesFileInitialContextFactory.java

```diff
@@ -164,6 +166,8 @@ protected ReadOnlyContext createContext(Map data, Hashtable environment)
 
     protected void createConnectionFactories(Map data, Hashtable environment) throws ConfigurationException
     {
+        resolveEncryptedProperties(environment);
```

Comment:
Shall we add a comment to this method even though you didn't create this method?